### ISSUE TITLE
Unable to import/open game save files from old VBA in VBA-M.

### ISSUE THREAD

**Issue by rmmariano** [visualboyadvance-m/visualboyadvance-m#839]
I'm trying to open a game save from an old version of VBA in my VBA-M.

I have tried to open using battery, state and snapshot files, however neither of the options work.

I'm using Pokemon Emerald and Fire Red roms.

### Expected behavior

VBA-M open old VBA save files.

### Actual behavior

VBA-M does not open old VBA save files.

### Steps to reproduce the behavior

I'm using a Win 7 virtual machine.

My games are [here](https://drive.google.com/file/d/1C0uuzZy41tCDvt3mlb6jwMUP2tC7TD1S/view?usp=sharing). Inside each folder there are VBA emulators, game save files and a folder with pictures illustrating the errors I have received.

Open one game folder, execute VBA-M, run the game rom and try to open the game save file using battery, state or snapshot.

The `pokemon_fire_red_error_01_battery.png` and `pokemon_emerald_error_01_battery.png` error pictures illustrate that when I try to import a battery file, the games freeze and they do not work anymore.

### ROM details

Which games are affected by the issue (please state if GBA or GB game): Pokemon Fire Red and Emerald.

Include screenshots of ROM Information (File > ROM Information):

![pokemon_emerald_rom_information](https://user-images.githubusercontent.com/10242509/116018842-4bbcf500-a619-11eb-973f-90a4e023e70c.png)

![pokemon_fire_red_rom_information](https://user-images.githubusercontent.com/10242509/116018847-4eb7e580-a619-11eb-8149-76d0a5be8168.png)

### Option details

Video driver (simple, opengl): simple, because opengl does not work, a white screen appears instead of the game using this option.

Sound driver (directsound, xaudio, openal, SDL): SDL

### Build details

Operating System (Windows, Mac, Linux (state distribution), etc.): Windows 7

Version of code (pre-built binary version, or commit ref, or just "master"): 2.1.3 and nightly

Thank you in advance.

**Comment by Squall-Leonhart** [comment]
_battery, _state and _snapshot are not suffixes used by VBA-M.

The importing of 1.8 or earlier states into vba-m, is not supported.

The sharing of copyrighted game files, invalidates your issue.

Closed.

**Comment by rmmariano** [comment]
Thank you for the help @Squall-Leonhart

I updated the issue to remove copyrighted game files. Thanks for the warning, I hadn't thought of this. 

I understood the issue of the difference in versions. However is there an approach that I could use? For example: try to import into an older version until I get to the most recent one? Like 1.9.0, 2.0.0, 2.1.0, etc. I looked for the old releases, but I didn't find an executable for Windows.
